package io.quarkus.rest.test.exception.resource;

public class ExceptionMapperCustomRuntimeException extends RuntimeException {
}
